_data/simple-icons.json

```
@@ -1485,6 +1485,11 @@
             "hex": "00AEEF",
             "source": "https://home.barclays"
         },
+        {
+            "title": "bard",
```

```suggestion
            "title": "Google Bard",
```

I would use this title, and it'd need moving to make it alphabetical. Don't forget to update the title inside the SVG too!
Otherwise, it's a perfect match!

---

Changes applied. Thank you for your feedback!